docs/eos-learn/blockchain-security.md

```diff
@@ -0,0 +1,8 @@
+---
+id: blockchain-security
+title: Blockchain Security
+sidebar_label: Security
+---
+## Blowing up vulnerabilities in Blockchain
```

Comment:
Blowing up is not the right term , seems its from a translation error (explotando = exploding = blowing up)

Reply:
Okay, I'm going to change it

Reply:
try this word: exploiting